Integrations/PaloAltoNetworks_Traps/PaloAltoNetworks_Traps.py

```diff
@@ -0,0 +1,543 @@
+''' IMPORTS '''
+
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import json
+import requests
+import copy
+import jwt
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS/PARAMS '''
+PARAMS = demisto.params()
+# Remove trailing slash to prevent wrong URL path to service
+SERVER = PARAMS['url'][:-1] \
+    if (PARAMS['url'] and PARAMS['url'].endswith('/')) else PARAMS['url']
+# Should we use SSL
+USE_SSL = not PARAMS.get('insecure', False)
+# How many time before the first fetch to retrieve incidents
+FETCH_TIME = PARAMS.get('fetch_time', '3 days')
+# Service base URL
+BASE_URL = SERVER + '/xapi/v1/'
+APPLICATION_ID = PARAMS.get('application_id')
+PRIVATE_KEY = PARAMS.get('private_key')
+# Headers to be sent in requests
+REQUEST_HEADERS = {
+    'Content-Type': 'application/json'
+}
+# Remove proxy if not set to true in params
+handle_proxy()
```

Comment:
please move handle proxy to the beginning of main()

Reply:
- [x] Fixed